{
  "summary": {
    "outcome": "Failed",
    "testsRan": 1,
    "passing": 0,
    "failing": 1,
    "skipped": 0,
    "passRate": "0%",
    "failRate": "100%",
    "skipRate": "0%",
    "testStartTime": "Mon Mar 13 2023 11:23:58 AM",
    "testExecutionTimeInMs": 670,
    "testTotalTimeInMs": 670,
    "commandTimeInMs": 117,
    "hostname": "https://akkodis41-dev-ed.develop.my.salesforce.com",
    "orgId": "00D0900000DYQKLEA5",
    "username": "raphael.houdry@fasha.com",
    "testRunId": "7070900003LI2gV",
    "userId": "0050900000BrlnPAAR"
  },
  "tests": [
    {
      "id": "07M09000003htxhEAA",
      "queueItemId": "709090000012GxaAAE",
      "stackTrace": "Class.OrderTriggerTest.az: line 67, column 1",
      "message": "System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, Price Book Not Set on Order: []",
      "asyncApexJobId": "7070900003LI2gVAAT",
      "methodName": "az",
      "outcome": "Fail",
      "apexLogId": "07L0900000BsQfKEAV",
      "apexClass": {
        "id": "01p0900000StH1hAAF",
        "name": "OrderTriggerTest",
        "namespacePrefix": null,
        "fullName": "OrderTriggerTest"
      },
      "runTime": 670,
      "testTimestamp": "2023-03-13T10:23:58.000+0000",
      "fullName": "OrderTriggerTest.az",
      "diagnostic": {
        "exceptionMessage": "System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, Price Book Not Set on Order: []",
        "exceptionStackTrace": "Class.OrderTriggerTest.az: line 67, column 1",
        "className": "OrderTriggerTest",
        "compileProblem": "",
        "lineNumber": 67,
        "columnNumber": 1
      }
    }
  ]
}